src/Modules/UngradedModules.jl

```diff
@@ -1531,6 +1551,31 @@ function cokernel(A::MatElem)
   return cokernel(map(A))
 end
 
+@doc Markdown.doc"""
+    image(F::FreeMod{T}, A::MatElem{T}) where T
+
+Return the subquotient $\im(A)$. The ambient free module is `F`.
+"""
+function image(F::FreeMod{T}, A::MatElem{T}) where T
+  @assert ncols(A) == rank(F)
+  return image(map(F,A))
+end
+
+@doc Markdown.doc"""
+    image(A::MatElem)
+
+Return the subquotient $\im(A)$. 
+
+!!! note
+
+    The ambient free module $R^m$ is constructed by the function and therefore not compatible with 
+    free modules $R^m$ that are defined by the user or by other functions. For compatibility,
+    use `image(F::FreeMod{T}, A::MatElem{T})`.
+"""
+function image(A::MatElem)
```

Comment:
Matrices have no "image". At least not an unambiguous one (do you consider the row span or the column span). We don't want to go down that road.